module/config/i18n/en-US.json

```diff
@@ -2060,6 +2060,10 @@
       "name": "Buy Weekly Roses",
       "help": "On week reset, roses will be available for purchase\nThe purchase is made in bulk wholsesale or none at all\nMinimum of 24K coin is required, if not enough subtask will be delayed to the next day and try again then"
     },
+    "BuyCake": {
+      "name": "Buy Weekly Cake Gift",
+      "help": "Cake gift purchase count is reset every week\nThe purchase is made in bulk wholsesale or none at all\nMinimum of 150 gems is required, if not enough subtask will be delayed to the next day and try again then"
```

Comment:
This is actually 210

Total of 5 can be bought and price at 30 but the price changes as you increase

This was also the case for roses, initially priced 1000 but actually each bump up increased the price even more.  Pretty naughty of them...